zerver/lib/actions.py

```diff
@@ -5183,11 +5186,10 @@ def do_get_user_invites(user_profile: UserProfile) -> List[Dict[str, Any]]:
     days_to_activate = settings.INVITATION_LINK_VALIDITY_DAYS
     active_value = confirmation_settings.STATUS_ACTIVE
     revoked_value = confirmation_settings.STATUS_REVOKED
-    
```

Comment:
did the previous commit introduce an empty whitespace here or something? :thinking: